media/css/newsletter/moznewsletter-subscribe.less

```diff
@@ -48,6 +48,7 @@
         padding: 10px 0 20px 180px;
         text-align: left;
         text-shadow: none;
+        height: 135px;
```

Comment:
While this fixes the bug for short strings, it breaks things for really long strings as it gives the element a fixed height. Please use `min-height: 135px;` instead. You may also need to [remove this line in home.scss](https://github.com/mozilla/bedrock/blob/master/media/css/mozorg/home/home.scss#L759) so `min-height` does not get overwritten.

Please just add another commit to your branch with this change, and github should pick up the changes here.